docs/7-infrastructure-configuration-management/7.1.4-terraform-providers.md

```diff
@@ -25,95 +25,87 @@ docs/7-infrastructure-configuration-management/7.1.4-terraform-providers.md:
       - Terraform
       - Go
     -
-      name: Implementing Dev, Ops, DevOps resource and datasource
-      description: Implement the CRUD operations for the Dev, Ops, and DevOps resources and datasources.
-      estMinutes: 300
+      name: Testing
+      description: Write comprehensive tests for the Engineer resource and datasource.
+      estMinutes: 120
       technologies:
       - Terraform
       - Go
     -
-      name: Challenge
-      description: Write comprehensive tests for the Engineer resource and datasource.
-      estMinutes: 120
+      name: Implementing Dev, Ops, or DevOps resource and datasource
+      description: Implement the CRUD operations for the Dev, Ops, and DevOps resources and datasources.
+      estMinutes: 300
```

Comment:
Hmmm I kind of like the exercise to cover `Engineer` and either `Dev` or `Ops`. That way you have to implement a composite resource. Is there a reason you didn't do it that way?

Reply:
The last exercise in the section is to implement one of the remaining resources. I'll update it to be precise about implementing either the Dev or Ops resource rather than the DevOps resource.

Reply:
👍 Thank you, I read `Challenge` as optional and I am sure others would as well. I would like apprentices to do one composite resource.